npm/webpack-dev-server/src/makeDefaultWebpackConfig.ts

```diff
@@ -106,6 +109,7 @@ export function makeCypressWebpackConfig (
     return {
       ...finalConfig,
       devServer: {
+        port: webpackDevServerPort,
```

Comment:
Again just `port` is okay, but this is more explicit

Reply:
Types problem without coercing to undefined